libs/commons/common2.mli

```diff
@@ -363,17 +332,15 @@ val run_hooks_action : 'a -> ('a -> unit) list ref -> unit
 type 'a mylazy = unit -> 'a
 
 (* emacs spirit *)
-val save_excursion : 'a ref -> 'a -> (unit -> 'b) -> 'b
```

Comment:
Can you point me to the other version of this? I think this version is used in pro.

Reply:
Nvm, found it in Common.